frontend/src/components/RatingTable.tsx

```diff
@@ -138,8 +138,9 @@ export const ratingTable: (def: RatingTableDef) => FC<RatingTableProps> = ({
           <Trans i18nKey="List order" />
           <Dropdown
             css={css}
-            title={t(TaskRatingSortingTypesToText(sorting.type.get()))}
-            maxLength={20}
```

Comment:
"complexity" didn't fit the limit, and since the possible title values are fixed, I removed the limit all together instead of guessing a better limit.